python/pyspark/sql/dataframe.py

```diff
@@ -678,13 +678,14 @@ def cache(self):
         return self
 
     @since(1.3)
-    def persist(self, storageLevel=StorageLevel.MEMORY_AND_DISK):
+    def persist(self, storageLevel=StorageLevel.MEMORY_AND_DISK_DESER):
```

Comment:
Now I see the confusion. In Scala, `MEMORY_AND_DISK` means `deserialized=true`, while in Python, `MEMORY_AND_DISK` means `deserialized=false`.